src/apps/popup/pages/token-details/token.tsx

```diff
@@ -45,41 +50,64 @@ type TokenInfoList = {
   value: string;
 };
 
-export const Token = () => {
-  const [token, setToken] = useState<TokenType | null>(null);
+type TokenProps = {
+  erc20TokensData: ContractPackageWithBalance[] | null;
+};
+
+export const Token = ({ erc20TokensData }: TokenProps) => {
```

Comment:
Why do we need this prop? You don't use it anywhere

Reply:
Good catch although that is fixed already just need to push